db/migrate/20150906173134_create_teams.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTeams < ActiveRecord::Migration
+  def change
+    create_table :teams do |t|
+      t.belongs_to :account
+      t.belongs_to :team
```

Comment:
this is for the team's parent, right? maybe name it `parent_team`? I could see `some_team.team` being hard to understand later. also, I don't see the `account` and `team` associations reflected in the Team model, unless it relies on some magic I'm not familiar with..